pilz_trajectory_generation/test/integrationtest_sequence_action_capability.cpp

```diff
@@ -60,7 +60,7 @@ class IntegrationTestBlendAction : public ::testing::Test
 
 protected:
   ros::NodeHandle ph_ {"~"};
-  actionlib::SimpleActionClient<pilz_msgs::MoveGroupBlendAction> ac_blend_{ph_, BLEND_ACTION_NAME, true};
+  actionlib::SimpleActionClient<pilz_msgs::MoveGroupSequenceAction> ac_blend_{ph_, BLEND_ACTION_NAME, true};
```

Comment:
shouldn't you also change the action client topic name (see comment on action server side above as well).

Reply:
Renamed.